src/web/app/src/components/AboutPage.tsx

```diff
@@ -107,7 +107,17 @@ const AboutPage = () => {
         you want to see it..
       </p>
 
-      <h2>What is a Planet?</h2>
+      <h2>What does Telescope do?</h2>
+      <p>
+        In essence, Telescope takes RSS blog post feeds, rebuilds those blog posts into HTML and
+        collects them in a single web page to display. It is able to handle various formatting, such
+        as code blocks or embedded videos. Telescope also collects data about those posts in the{' '}
+        <a href="https://api.telescope.cdot.systems/v1/status/">dashboard</a>. Blog posts on
+        Telescope come from contributors to Telescope. Therefore, you get to follow the
+        website&quot;s development on the website itself!
+      </p>
+
+      <h2>It all starts with a Planet</h2>
```

Comment:
What about "started" vs. "starts", since we don't use the planet anymore, but it inspired the work.

Reply:
Changed.